tests/test_board.py

```diff
@@ -1,60 +1,142 @@
 import unittest
+
 from rookpoly import Board
-from rookpoly import Polynomial
+from rookpoly.polynomial import Polynomial
 
 
 class TestBoard(unittest.TestCase):
-
     empty_boards_limit = 7
 
     def setUp(self):
         self.empty_boards = [Board(['0' * i for _ in range(i)]) for i in range(self.empty_boards_limit)]
-        self.board_1 = Board(["110", "100", "000", "001"])
-        self.board_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+        self.board_global_1 = Board(["110", "100", "000", "001"])
```

Comment:
Why the rename is needed?

Reply:
Because later in the tests local boards are also used

Reply:
This is a nitpick and you can leave it as is, but `self.board_1` would not be shadowing `board_1` anyway

Reply:
I understand, but since board data is declared before all tests, it is better to indicate their function in their name